django_large_image/templates/django_large_image/_include/colors.html

```diff
@@ -18,15 +18,90 @@
     @click="controls = ! controls"
   >
     Controls
-    <i class="ri-equalizer-line text-lg font-normal" x-show="!controls"></i>
-    <i class="ri-close-line text-lg font-normal" x-show="controls"></i>
+    <i class="ri-equalizer-line" x-show="!controls"></i>
+    <i class="ri-close-line" x-show="controls"></i>
   </button>
 
-  <div class="absolute bottom-full pb-3 right-0" x-show="controls">
-    <div class="card bg-gray-100 bg-opacity-[.90] border border-gray-300 min-w-max !rounded-lg shadow-lg">
-      <div class="card-body">
-        <div class="grid grid-cols-2 gap-8">
-          <div class="controls flex flex-col gap-6">
+  <div
+    class="absolute bottom-full pb-3 right-0"
+    x-show="controls"
+  >
+    <div class="card bg-slate-100 bg-opacity-[.85] min-w-max !rounded-lg shadow-lg">
+      <div
+        class="card-body !p-0"
+        x-data="{ customizeDisplay: false }"
+      >
+        <div
+          class="grid gap-4"
+          :class="{ 'grid-cols-2' : customizeDisplay }"
+        >
+          <div
+            x-show="customizeDisplay"
+            x-data="{ configure: false }"
+            class="row-span-full w-56"
+          >
+            <div
+              class="bg-secondary flex flex-col h-full text-slate-900 justify-between p-4 text-secondary-content"
+              x-data="{ items: [
+                { frame: 'Frame 1', bands: ['Band 1', 'Band 2', 'Band 3'] },
+                { frame: 'Frame 2', bands: ['Band 1', 'Band 2'] }
+              ]}"
+            >
+              <div>
+                <ul class="menu menu-compact menu-secondary p-1 rounded gap-2">
+                  <template x-for="item in items">
+                    <li tabindex="0">
+                      <a class="font-bold" x-text="item.frame"></a>
+                      <ul class="!block !static !p-0 rounded">
+                        <template x-for="band in item.bands">
+                          <li>
+                            <a class="flex gap-1" :class="{ 'active' : configure }" @click="configure = ! configure">
+                              <i class="ri-arrow-right-line"></i>
+                              <span x-text="band"></span>
+                            </a>
+                          </li>
+                        </template>
+                      </ul>
+                    </li>
+                  </template>
```

Comment:
Notice the `x-for="item in items"` and `x-for="band in item.bands"`. These are the for-loops mentioned in the comments below. Again, this will most likely be different, but if you can use the AlpineJS logic, feel free. Hopefully it doesn't cause too much of a headache.